src/types/reduxTypes/areaSpecializationTypes.ts

```diff
@@ -113,3 +113,29 @@ export interface INotificationsProps {
   name: string
   applicationUuid: string
 }
+
+export interface IUsersList {
+  address?: string
+  canDeactivate: boolean
+  email: string
+  imageURL: string | null
+  name: string
+  role: string
+  uuid: string
+}
+
+export interface IDeactivateUserProps {
+  uuid: string
+  params: IFilterUserListProps
+}
+
+export interface IFilterUserListProps {
+  page: number
+  key?: string
+  value?: string
+}
+
+export enum UserActivate {
+  Deactivated = 'Deactivated',
+  Activated = 'Activated',
```

Comment:
Deactivated is some kind of user role so it should be listed in userRoles enum

Reply:
done